README.md

```diff
@@ -10,6 +10,14 @@ _In fact, there is nothing substantial to see here yet_
 
 Join our [#radicle](https://webchat.freenode.net/?channels=radicle) IRC channel for development updates.
 
+## Build
+
+To build this, besides a rust build environment, you'll need the following libraries and tools (these are the package names on Ubuntu or Debian):
+
+* `libssl-dev` (ssl library)
+* `nettle-dev` (low level crypto library)
+* `m4`, `llvm`, `libclang`, `libclang-dev` (dev tools and libs)
```

Comment:
I think `pkgconfig` could be needed as well

Reply:
Done.